Sources/CaptureSwiftyBeaverIntegration/CaptureSwiftyBeaverLogger.swift

```diff
@@ -0,0 +1,52 @@
+import Capture
+import SwiftyBeaver
+
+/// The wrapper around Capture SDK logger that conforms to `BaseDestination` protocol from `SwiftyBeaver`
+/// library and can be used as a drop-in solution for forwarding `SwiftyBeaver` logs to bitdrift
+/// Capture SDK.
+final class CaptureSwiftyBeaverLogger: BaseDestination {
+    override func send(
+        _ level: SwiftyBeaver.Level,
+        msg: String,
+        thread: String,
+        file: String,
+        function: String,
+        line: Int,
+        context: Any? = nil
```

Comment:
shouldn't we also fwd the rest of these ^ fields to our logger?

Reply:
what about thread?

Reply:
For threads we could just add it as one of the fields no? I don't see the value on having to wait until we implement our own approach if they already provide it